### ISSUE TITLE
Listeria rendering centuries as the next century

### ISSUE THREAD

**Issue by pere-prlpz** [magnusmanske/listeria_rs#132]
When dates are set as centuries in Wikidata, Listeria used to render them properly. However, now it's adding one century to all centuries in cawiki. Example: https://ca.wikipedia.org/w/index.php?title=Llista_de_top%C3%B2nims_d%27Horta-Guinard%C3%B3&curid=2033256&diff=34076841&oldid=34043242

I've checked some of the changed dates and the date in Wikidata matches the previous version of the list.

**Comment by prototyperspective** [comment]
Had the same problem on Wikimedia Commons and it's fixed there now, maybe it's fixed now at WP as well?

**Comment by pere-prlpz** [comment]
Yes. I've checked just a few cases but they seem to work fine. Thanks to whoever fixed it.

**Comment by magnusmanske** [comment]
Closing — confirmed fixed in the thread. Century precision is handled in `Reference::format_date_by_precision` (`src/reference.rs`) with unit tests for century, decade and millennium precision, including negative years.
